_includes/master/endpointport.md

```diff
@@ -0,0 +1,13 @@
+An EndpointPort associates a name with a particular TCP/UDP port of the endpoint, allowing it to
+be referenced as a named port in [policy rules]({{site.baseurl}}/{{page.version}}/reference/calicoctl/resources/policy#entityrule).
```

Comment:
Does this link render correctly with a space (apparently) between the `]` and the `(` ?  It's a useful TIL if so!

Reply:
No, there's no space, it's just how github renders it if the window is too small.